src/ServiceControl.AcceptanceTests/Recoverability/Groups/When_a_group_is_archived.cs

```diff
@@ -62,6 +62,59 @@ await bus.SendLocal<MyMessage>(m => m.MessageNumber = 2)
                 .Run();
         }
 
+        [Test]
+        public async Task All_archived_messages_should_be_grouped()
```

Comment:
Looks good. Should we have another test that also exercises the `archive/groups/id/{groupId}` route?

Reply:
Added